mk_verificator/global_config.yaml

```diff
@@ -5,11 +5,11 @@ admin_project_id: 1111
 url: ""
 url_v3: ""
 endpoint_type: internalURL
-image_ref: b994881e-773a-4227-94cb-1082015a0930
+image_ref: 1111
```

Comment:
I suppose better to determine uuid in the format:
11111111-1111-1111-1111-111111111111